### PR TITLE
russian locales

### PR DESCRIPTION
**Pull request by Telonor**

### CODE REVIEW

File: django_password_strength/locale/ru/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (aj-may) [inline django_password_strength/locale/ru/LC_MESSAGES/djangojs.po]: Can we update the boilerplate above?

File: django_password_strength/locale/ru/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (aj-may) [inline django_password_strength/locale/ru/LC_MESSAGES/django.po]: Can we update this boilerplate ^

**Comment by aj-may** [comment]
Just made small comments about the comment block at the top of the new translation files.  Other than that, it looks great.  Thanks for the contribution.

**Comment by Telonor** [comment]
I've made changes if this was a point. But u can update this, if u need.

**Comment by aj-may** [comment]
I pulled your commits into a new branch and rebased out the `.mo` files.  The changes are in this PR #8 

Closing this PR in favor of #8

**Pull request closed without merging**